insights/combiners/httpd_conf.py

```diff
@@ -235,9 +237,64 @@ def get_active_setting(self, directive, section=None):
         if section is not None:
             if values_list:
                 for i, val in enumerate(values_list):
+                    # From each section, preserve only the last ParsedData
+                    # {(section, ""): [ParsedData, ParsedData]} ---> ParsedData
                     values_list[i] = val.values()[0][-1]
                 return values_list
             return []
         else:
             if values_list:
                 return values_list[-1]
+
+    def get_section_list(self, section):
+        """
+        Returns the specified sections.
+
+        Parameters:
+            section (str): The section to look for, e.g. "Directory"
+
+        Returns:
+            (list of dict): List of tuples, each tuple has three elements - the
+                first being a tuple of the section and section name, the second
+                being the file name of the file where that section resides,
+                the third being the full file path of the file.
```

Comment:
@jsvob the indentation on this should match the first line (list of dict), you can see the problem in the rendered output.  Also, is it a list of dict or list of tuples?

Reply:
@bfahr Good point, it's a list of tuples. Fixed. I tried to compile the documentation according to https://github.com/RedHatInsights/insights-core/wiki and it errored out (used to work for me before but I can't rule out I didn't miss something during the recent insights transition to 3.0)), so I hope it'll be fine.